### PR TITLE
[docs] Correct usage dataStream API for release-2.0

### PR DESCRIPTION
**Pull request by mincwang**
Correct usage dataStream API for release-2.0 branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,14 +71,18 @@ import com.ververica.cdc.connectors.mysql.MySqlSource;
...
+    Properties debeziumProperties = new Properties();
+    debeziumProperties.put("snapshot.locking.mode", "none");// do not use lock
+    SourceFunction<String> sourceFunction = MySQLSource.<String>builder()
+            .hostname("yourHostname")
+            .port(yourPort)
+            .databaseList("yourDatabaseName") // set captured database
+            .tableList("yourDatabaseName.yourTableName") // set captured table
+            .username("yourUsername")
+            .password("yourPassword")
+            .deserializer(new StringDebeziumDeserializationSchema()) // converts SourceRecord to JSON String
```
Comment (leonardBang) [inline README.md:83]: no JSON here

File: docs/content/about.md [hunk 2]
```diff
@@ -88,14 +88,18 @@ import com.ververica.cdc.connectors.mysql.MySqlSource;
...
+    Properties debeziumProperties = new Properties();
+    debeziumProperties.put("snapshot.locking.mode", "none");// do not use lock
+    SourceFunction<String> sourceFunction = MySQLSource.<String>builder()
+        .hostname("yourHostname")
+        .port(yourPort)
+        .databaseList("yourDatabaseName") // set captured database
+        .tableList("yourDatabaseName.yourTableName") // set captured table
+        .username("yourUsername")
+        .password("yourPassword")
+        .deserializer(new StringDebeziumDeserializationSchema()) // converts SourceRecord to JSON String
```
Comment (leonardBang) [inline docs/content/about.md:100]: no JSON here

File: docs/content/connectors/mysql-cdc.md [hunk 3]
```diff
@@ -370,14 +370,19 @@ import com.ververica.cdc.connectors.mysql.MySqlSource;
...
+    Properties debeziumProperties = new Properties();
+    debeziumProperties.put("snapshot.locking.mode", "none");// do not use lock
+    SourceFunction<String> sourceFunction = MySQLSource.<String>builder()
+        .hostname("yourHostname")
+        .port(yourPort)
+        .databaseList("yourDatabaseName") // set captured database
+        .tableList("yourDatabaseName.yourTableName") // set captured table
+        .username("yourUsername")
+        .password("yourPassword")
+        .deserializer(new StringDebeziumDeserializationSchema()) // converts SourceRecord to JSON String
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc.md:382]: ditto

File: README.md (hunk 1 above)
Comment (mincwang) [reply README.md]: my fault

**Review by leonardBang — APPROVED** [review]
Thanks @mincwang for the update, LGTM

**Pull request merged**